app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -41,6 +41,7 @@ CONFIG_DMA_DW_LLI_POOL_SIZE=50
 CONFIG_INTEL_MODULES=y
 CONFIG_LIBRARY_MANAGER=y
 CONFIG_LIBRARY_AUTH_SUPPORT=y
+CONFIG_LIBRARY_PIPELINE_FORCE_MIN_LIFETIME=100
```

Comment:
Sorry, not following commit message, it not clear where the CPC calculation is transitioning from/to,  i.e. CPC at pipeline start/stop is start and end point of this commit, pls also say why current method is broken. i.e. what use case.